home/fixtures/test.json

```diff
@@ -0,0 +1,122 @@
+[
```

Comment:
Why do we have this file? Is it used in test?

These are a pain to maintain. Better create content programatically.

Reply:
This is not used currently. I created this a while ago when I was trying out fixtures. I didn't delete it because I thought we may need it in future. I'll delete it.